react-app/src/native/components/leagues/Schedule.js

```diff
@@ -59,41 +47,23 @@ export default class Schedule extends React.Component {
             teamTable.data.push([team.numInLeague, team.name]);
         });
 
+
         if (league == null || league.isFetching) return <Loading />
 
         return (
             <Container>
-                <Content padder>
-                    <Card>
-                        <CardItem header>
-                            <Text>{leagueName}</Text>
-                        </CardItem>
-                        <CardItem cardBody style={{ padding: 10 }}>
-                            <Table style={{ flex: 1, marginBottom: 10 }} borderStyle={{ borderWidth: 0, borderColor: "transparent" }}>
-                                <Row
-                                    flexArr={flexArr}
-                                    data={teamTable.header}
-                                    style={styles.header}
-                                    textStyle={styles.headerText}
-                                />
-                                {
-                                    teamTable.data.map((rowData, index) => (
-                                        <Row
-                                            key={index}
-                                            flexArr={flexArr}
-                                            data={rowData}
-                                            style={[styles.row, index % 2 == 1 && { backgroundColor: '#e6e6e6' }]}
-                                            textStyle={styles.text}
-                                        />
-                                    ))
-                                }
-                            </Table>
-                        </CardItem>
-                    </Card>
+                <Content>
 
                     {
                         league != null && !league.isFetching &&
```

Comment:
this league != null && league.isFetching check will never return false. If it were false the render function would return the loading screen on line 51